templates/genie.yaml

```diff
@@ -69,6 +72,27 @@ Resources:
         InstanceTypes:
           - optimal
         DesiredvCpus: 8
+  DevBatchComputeEnvironment:
```

Comment:
hmm.  wondering what the goal is here?  did you really want to deploy batch job into the same stack with shared resources? or did you want to keep them separate?  if you want to keep them separate you may want to re-use the template as-is and just deploy another 'dev' batch instance ?

Reply:
Eventually I will use the new batch environment as production, but I don't want the auto validation to run with this environment right now.  I will manually submit jobs to the compute environment.